arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,227 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "SqlManagementClient",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete Databases, Servers and related resources.",
+        "version": "2015-05-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json",
+        "text/json"
+    ],
+    "produces": [
+        "application/json",
+        "text/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/securityAlertPolicies/Default": {
+            "put": {
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_CreateOrUpdateServerSecurityAlertPolicy",
+                "description": "Creates or updates an Azure SQL Server security alert policy.",
```

Comment:
@yaakoviyun 
"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/securityAlertPolicies//Default"
Based on the path definition it seems the put and get operations are for creating/updating and getting the *default* security policy only. This should be clearly mentioned in the description to avoid any confusion to the end user. The description will become the documentation in the generated code. There are plans to keep the msdn API documentation upto date using the swagger spec. Hence, what you mention in the description becomes even more important.